src/game.cpp

```diff
@@ -13859,6 +13857,10 @@ void game::add_artifact_messages(std::vector<art_effect_passive> effects)
         case AEP_SICK:
             add_msg(m_bad, _("You feel unwell."));
             break;
+
+        case AEP_SMOKE:
+            add_msg(m_warning, _("The %s begins to smoke..."));
```

Comment:
The '%s' requires another parameter (a c-string), currently this invokes undefined behavior. Add the name of the item as parameter here, like it was done in the original code.

Btw. please compile your code before pushing it to let the compiler check for errors.